### ISSUE TITLE
Not being able to set radio code

### ISSUE THREAD

**Issue by Pergor** [michail-nikolaev/task-force-arma-3-radio#1251]
**Arma 3 Version:** `1.68`
**CBA Version:** `3.3.0.170502`
**Task Force Radio Version:** `1.-1.0.235`

**Mods:**
- `@CBA_A3`
- `@task_force_radio`

**Description:**
- It's currently not possible to set radio codes before tfar's postInit is over. It works fine for example  for TFAR_defaultFrequencies_sr_independent, but the radio code is somehow overwritten at a later time.
Additionally it's not really clear which variable is actually used for SR radios: TFAR_freq_sw_* or TFAR_freq_sr_*

**Steps to reproduce:**
- Call this on preInit:
_settingsSRWest = false call TFAR_fnc_generateSwSettings;
_settingsSRWest set [4, "_test"];
TFAR_freq_sw_west = _settingsSRWest;
TFAR_freq_sr_west = _settingsSRWest;
-> TFAR_freq_sr_west select 4 will revert to "_bluefor", while TFAR_freq_sw_west select 4 will be "_test".

**Comment by dedmen** [comment]
https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/1.0/addons/core/functions/server/fnc_serverInit.sqf#L78
So either `TFAR_defaultFrequencies_sr_west` (Which is what most people need) or setting the whole `TFAR_freq_sr_west`

**Comment by Pergor** [comment]
Ah, I see. Got it working, thanks!

But then there's a follow up problem: missionNamespace getVariable "tf_SIDE_radio_code" seems not to be updated with the current value for TFAR_freq_lr_* select 4.

**Comment by dedmen** [comment]
`tf_SIDE_radio_code` doesn't relate to TFAR_freq stuff. The freq defines the radio code of the specific radio.
`tf_SIDE_radio_code` is just what's used inside configs and denotes the default encryptions

**Comment by Pergor** [comment]
Yeah, but if I change TFAR_freq_lr_* select 4 I can communicate with other SR- and LR-radios using the specific TFAR_freq_lr_*. Except for vehicles that revert to the variable they get from tf_SIDE_radio_code.
That way setting TFAR_freq_lr_* select 4 has no impact on vehicles.

**Comment by dedmen** [comment]
So vehicles ignore the radio code set in TFAR_freq_lr ?

**Comment by Pergor** [comment]
Yes, it seems so.
